src/editors/sharedComponents/SelectionModal/Gallery.jsx

```diff
@@ -25,12 +23,8 @@ export const Gallery = ({
   showIdsOnCards,
   height,
   isLoaded,
-  // injected
-  intl,
 }) => {
-  if (!show) {
-    return null;
-  }
```

Comment:
I removed this because I didn't see a good reason this should be a prop. If a parent component wants to hide this component it can just not render it instead of passing this prop and having it not render itself.